src/SDKs/DeploymentManager/DeploymentManager.Tests/Management.DeploymentManager.Tests.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DeploymentManager.Tests</PackageId>
+    <Description>DeploymentManager.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.DeploymentManager.Tests</AssemblyName>
+    <Version>0.9.0-preview</Version>
```

Comment:
Please set this as `1.0.0`

Reply:
Keeping this the same like we discussed as we're in preview.

Reply:
Setting this to 1.0.0 as this is a test project.